specification/postgresql/resource-manager/readme.md

```diff
@@ -39,11 +39,31 @@ tag: package-flexibleserver-2021-06
 tag: package-2020-01-01
 ```
 
+
+### Tag: package-flexibleserver-2022-03-preview
```

Comment:
We need to add this new version to other readme.<language>.md files

e.g.
`
### Tag: package-flexibleserver-2022-01-preview and <language_name>

These settings apply only when `--tag=package-flexibleserver-2022-03-preview --<language_name>` is specified on the command line.
Please also specify `--<language_name>-sdks-folder=<path to the root directory of your azure-sdk-for-<language_name>clone>`.

``` yaml $(tag) == 'package-flexibleserver-2022-01-preview' && $(<language_name>)
namespace: <language_namespace>
output-folder: <language_output_folder>
```
`